frontend/src/features/mapSideBar/SidebarContents/AssociatedLandForm.tsx

```diff
@@ -422,16 +431,56 @@ const AssociatedLandForm: React.FC<IAssociatedLandForm> = (props: IAssociatedLan
         <SteppedForm<IAssociatedLand>
           // Provide the steps
           steps={[
-            { route: 'ownership', title: 'Land Ownership', completed: false, canGoToStep: true },
+            {
+              route: 'ownership',
+              title: 'Land Ownership',
+              completed: false,
+              canGoToStep: true,
+              validation: {
```

Comment:
each step that needs validation requires the new validation prop.

Reply:
note that the prop does nothing unless the validateCurrentStep method is called.